samples/boards/ti/cc1352r_sensortag/sample.yaml

```diff
@@ -0,0 +1,6 @@
+sample:
+  name: TI CC1352R SensorTag Demo
+tests:
+  sample.boards.ti.cc1352r_sensortag:
+    build_only: true
```

Comment:
why is this flagged as build_only?  Does it not actually run on the board?

Reply:
I don't actually know. TBH, I was using `samples/boards/ti/cc13x2_cc26x2/system_off/sample.yaml` as a template. @vanti - any idea? I can only only speculate that maybe it has something to do with hardware-in-the-loop CI maybe?

Reply:
I think it can be safely removed given that the `up_squared/gpio_counter` sample does not have that line.

Reply:
@cfriedt I made "system_off" build-only because the sample enables power management and shuts down the entire system. I have seen sometimes when it is in that state, OpenOCD would fail to flash the device on the next attempt (bad in the context of running sanitycheck), so I excluded the sample from being run. I don't think you would have the same issue here.

Reply:
@vanti  - Ok, I see. I removed the line. Thanks for clarifying